test/lib/meetup_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+require_relative '../support/meetups/sample_api_responses'
+
+class MeetupTest < ActiveSupport::TestCase
+  test "some test" do
+    Meetup.stubs(:get).with("/pro/operationcode/groups", options).returns(group_endpoint_response)
```

Comment:
You'll note that `HTTParty`'s `get` method is in fact a class method on `Meetup`.  

This line says: 

_In the tests,
whenever you are going to call that `self.class.get` method, 
with these exact parameters: `"/pro/operationcode/groups", options`,
instead of using `HTTParty`,
just return whatever `group_endpoint_response` returns._